File: dmytryshyn/Lab3/Lab3/Lab3/Classes/Items/Abstractions/Item.cs
using Lab3.Builders;

namespace Lab3.Classes.Items.Abstractions
{
    abstract public class Item
    {
        public int ID { get; }
        public double Weight { get; }
        public int Count { get; }
        public int ContainerID { get; }

        public Item(int id, double weight, int count, int containerId)
        {
            ID = id;
            Weight = weight;
            Count = count;
            ContainerID = containerId;
        }

        public abstract double GetTotalWeight();
    }
}

File: mukha/lab_3/PortManagementSystem/items/item.py
class Item:
    def __init__(self, config):
        self.id = config['ID']
        self.weight = config['weight']
        self.count = config['count']
        self.container_id = config['containerID']

    def to_dict(self):
        return {
            'ID': self.id,
            'weight': self.weight,
            'count': self.count,
            'containerID': self.container_id
        }
